loadable_apps/loadable_sample/wifiapp/binary_update.c

```diff
@@ -402,17 +438,23 @@ static void binary_update_unregister_state_changed_callback(void)
 		fail_cnt++;
 		printf("Unregister state changed callback FAIL, ret %d\n", ret);
 	}
+
+	return ret;
 }
 
 static void binary_update_same_version_test(void)
 {
-	int ret;
+	int ret;	
```

Comment:
This function is tightly related to result of other function, in this case you had better change function type to int to notify to user clearly. but it just sample app you can ignore this comment.

Reply:
Each functions which have a name binary_update_XXX_test are independent. So it doesn't have to return result I think.